chatwoot/chatwoot
# fix: issue of translation for `TimeAgo.vue`

# fix: issue of translation for `TimeAgo.vue`

## Description

This PR intents to solve issue #7862 , which refers to a lake of translations on `TimeAgo.vue` file. 
Fixes #7862 

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

- Run the app locally
- Go to a single conversation
- Check the last activity 

## Checklist:

- [] I have commented on my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
